common/reviews/api/gulp-core-build.api.md

```diff
@@ -131,6 +131,7 @@ export interface IBuildConfig {
     libES6Folder?: string;
     libESNextFolder?: string;
     libFolder: string;
+    maxBuildTime: number;
```

Comment:
Shouldn't that be maxBuildTime **?** : number;

Reply:
Because we support partially updating the build config, it's hard to set it back to `undefined`. To disable the timeout, just set it to `0`.